Detectors/TRD/workflow/src/TRDGlobalTrackingSpec.cxx

```diff
@@ -79,21 +84,32 @@ void TRDGlobalTracking::init(InitContext& ic)
 
   mTracker->PrintSettings();
 
+  auto& elParam = o2::tpc::ParameterElectronics::Instance();
+  auto& gasParam = o2::tpc::ParameterGas::Instance();
+  mTPCTBinMUS = elParam.ZbinWidth;
+  mTPCVdrift = gasParam.DriftV;
+  mTracker->SetTPCVdrift(mTPCVdrift); // possibly a value from the CCDB should be used instead?
+
```

Comment:
I think it is better to move this in e.g. `updateTimeDependentParams` method called from `run()`, since in the real run these parameters should be queried for particular TF timestamp. 
I've already started to do this in different workflows.

Reply:
Sure that I have changed. Although the proper CCDB interface is still needed. I guess the TPC time bin granularity will be constant, but vDrift changes regularly